pkg/store/prometheus.go

```diff
@@ -496,9 +500,47 @@ func (p *PrometheusStore) LabelValues(ctx context.Context, r *storepb.LabelValue
 		return &storepb.LabelValuesResponse{Values: []string{l}}, nil
 	}
 
-	vals, err := p.client.LabelValuesInGRPC(ctx, p.base, r.Label, nil, r.Start, r.End)
-	if err != nil {
-		return nil, err
+	var (
+		sers []map[string]string
+		err  error
+	)
+
+	lvc := false // LabelValuesCall
+	vals := []string{}
+	v := p.promVersion()
+
+	version, err1 := semver.Parse(v)
+	baseVer, err2 := semver.Make("2.24.0")
```

Comment:
You can make it a global variable and we know it doesn't throw an error

Reply:
Why should we make it a global variable? 🤔  We can still keep `baseVer` a local variable and don't check for an error while declaring and initializing it.

Reply:
For performance. If you call `LabelValues` 1k times, then `semver.Make("2.24.0")` will be called 1k times. If you make it a global variable then it is parsed only once.